.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,141 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+  "name": "Python 3",
+  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+  // TODO: Bullseye has glibc 2.31 and cadquery-ocp@7.7.2 needs 2.35
+  "image": "mcr.microsoft.com/devcontainers/python:1-3.10-bookworm",
+  // Features to add to the dev container. More info: https://containers.dev/features.
+  "features": {
+    "ghcr.io/deep-space-cartel/devcontainers-features/apt:latest": {
+      // TODO: cmake is a dependency of Python package "pycairo"
+      // TODO: as well as those: libcairo2-dev pkg-config python3-dev // https://pycairo.readthedocs.io/en/latest/getting_started.html
+      // ffmpeg libsm6 libxext6 to fix ImportError: libGL.so.1: cannot open shared object file: No such file or directory
+      // openscad: is needed for "pc inspect :test"
+      "packages": "shellcheck bash-completion cmake libcairo2-dev pkg-config python3-dev ffmpeg libsm6 libxext6 openscad" // shfmt
```

Comment:
This is "editable" mode. There are couple options for long term:

- [ ] Switch to `Dockerfile` based Dev Containers and add `RUN apt-get ...` there.
  - This is a bit more flexible but provisioning slow and not reproducible.
- [ ] Build new "base" image for `partcad` development env.
  - This is a bit less flexible but provisioning fast and reproducible.